app/controllers/login/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+exports.logout = require('./logout-controller')
```

Comment:
I think we should use `module.exports` instead of `exports` as we always return `module.exports` from the module